core/src/main/java/org/ehcache/events/StoreEventListener.java

```diff
@@ -25,14 +26,16 @@
    *
    * @param key the <code>key</code> of the mapping being evicted
    * @param valueHolder the {@link Store.ValueHolder} being evicted
+   * @param timeSource the associated {@link TimeSource}
    */
-   void onEviction(K key, Store.ValueHolder<V> valueHolder);
+  void onEviction(K key, Store.ValueHolder<V> valueHolder, TimeSource timeSource);
```

Comment:
Let's work in order to not have to do this API change, which makes little sense.
From usage, I see that if the `CacheEventNotificationServiceImpl` had access to a `TimeSource` the param is no longer needed. My IDE tells me this is the only usage.